backend-shared/config/db/utils_test.go

```diff
@@ -84,3 +88,52 @@ func TestIsEmptyValues(t *testing.T) {
 		})
 	}
 }
+
+var _ = Describe("Utils Test", func() {
+	Context("Utils test", func() {
```

Comment:
Can change this to `Context("Ensure ValidateFieldLength reports database rows that are too long", func() {`, just to give more context to the test.